context.go

```diff
@@ -25,10 +25,18 @@ func SetAutoescape(newValue bool) {
 //	{{ myfunc("test", 42) }}
 //	{{ user.name }}
 //	{{ pongo2.version }}
-type Context map[string]any
+type Context struct {
+	*SyncMap[string, any]
```

Comment:
Why struct?

Reply:
Because along with syncMap functionality, `Context` is also implementing two methods - `checkForValidIdentifiers` and `Update`. Can those be handled without embedding ?